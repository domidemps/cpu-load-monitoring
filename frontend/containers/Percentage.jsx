/** @jsx jsx */
import {css, jsx} from '@emotion/core'
import Paper from './Logs'

export default function Percentage() {
  return <Paper elevation={1} />
}
