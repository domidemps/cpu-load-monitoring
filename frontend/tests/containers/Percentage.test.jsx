import React from 'react'
import configureStore from 'redux-mock-store'
import thunk from 'redux-thunk'
import {Provider} from 'react-redux'
import renderer from 'react-test-renderer'

import Percentage from '../../containers/Percentage'

describe('Percentage', () => {
  let store

  it('renders correctly the current average CPU load', () => {
    store = configureStore([thunk])({
      load: {
        averageLoad: 0.6,
      },
    })
    expect(
      renderer.create(
        <Provider store={store}>
          <Percentage />
        </Provider>,
      ).toJSON(),
    ).toMatchSnapshot()
  })
})
