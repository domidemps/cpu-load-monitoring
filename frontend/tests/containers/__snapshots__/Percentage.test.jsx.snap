// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Percentage renders correctly the current average CPU load 1`] = `
<div
  className="MuiPaper-root css-r93fnb-Percentage MuiPaper-elevation5 MuiPaper-rounded"
>
  <div
    className="css-qy1xbs-Percentage"
  >
    <h5
      className="MuiTypography-root css-1gri5ec-Percentage MuiTypography-h5"
    >
      Average CPU load
    </h5>
    <h1
      className="MuiTypography-root css-1h9ee3p-Percentage MuiTypography-h1"
    >
      60%
    </h1>
  </div>
</div>
`;
